v2/prixite/pages/_app.js

```diff
@@ -0,0 +1,30 @@
+import "../styles/globals.css";
```

Comment:
@aftab76ahmad76 why the name of this file begins with underscore "_"?

Reply:
It's by default in next js